config/locales/dotiw/en.yml

```diff
@@ -2,4 +2,4 @@
 en:
   datetime:
     dotiw:
-      two_words_connector: and
+      two_words_connector: " and "
```

Comment:
I think our YAML normalization will trim/strip this 😬 

I worked around this by adding the spaces in Ruby in a different file: https://github.com/18F/identity-idp/blob/master/app/presenters/account_reset/pending_presenter.rb#L17

Reply:
> I think our YAML normalization will trim/strip this 😬
> 
> I worked around this by adding the spaces in Ruby in a different file: https://github.com/18F/identity-idp/blob/master/app/presenters/account_reset/pending_presenter.rb#L17

Hm! I don't see those normalizations happening locally, though I could see how that'd be an interesting challenge to deal with.

The current failures actually seem to be caused by a doubling-up of the spaces for specs associated with the file you mention. Should be as easy to fix as removing said workaround.

Not sure if it's something that changed in a more recent upgrade of the `i18n-tasks` dependency. It would seem to me as overly-opinionated of a normalization tool to forbid meaningful whitespace like this 🤔

Reply:
Another possibility is that previous attempts hadn't included the `"` quoting, in which case I'd expect the trimming behavior you describe to be applied to any excess whitespace in the YAML file.

Reply:
I don't remember the rationale or goal anymore... but we do *some* trailing whitespace trimming in our class:

https://github.com/18F/identity-idp/blob/master/lib/yaml_normalizer.rb#L50-L55
https://github.com/18F/identity-idp/blob/master/spec/lib/yaml_normalizer_spec.rb#L64-L73

Reply:
Ah, I'd missed the additional normalization we do beyond `i18n-tasks`. Yeah, I now see the problem you mention.

Thoughts:

- Should we avoid the trimming altogether? Or maybe at least apply trimming on the dumped output, rather than on the individual values?
- I've found some examples of using [Psych](https://github.com/ruby/psych) combined with its [`parse_stream`](https://docs.ruby-lang.org/en/2.5.0/Psych.html#method-c-parse_stream) and AST node [`style` attribute + `SINGLE_QUOTED` / `DOUBLE_QUOTED` constants](https://docs.ruby-lang.org/en/2.5.0/Psych/Nodes/Scalar.html) to have better control on the parsing and output styles ([example](https://stackoverflow.com/a/47556711)). Could consider for a refactor to give us better control to avoid trimming quoted values.

In the meantime, I think I'll revert back to a workaround like you originally proposed, since it seems those ideas may take a bit more effort to implement than I can afford now.

Reply:
Updated in c91c509 to pass option everywhere we use `distance_of_time_in_words`.

Reply:
Yeah I think a great alternative would be trim but only for unquoted strings or something but agree, definitely better to tackle in a separate PR